README.md

```diff
@@ -3,20 +3,14 @@
 This repository contains the Puppet modules for WSO2 Identity Server and the profiles related to Identity Server Analytics.
 
 ## Quick Start Guide
-1. Download an updated wso2is-5.8.0.zip or wso2is-analytics-5.8.0.zip according to your preferred deployment pattern and copy it to the `<puppet_environment>/modules/common/files` directory in the **Puppetmaster**.
+1. Download an updated wso2is-5.8.0.zip or wso2is-analytics-5.8.0.zip according to your preferred deployment pattern and copy it to the `<puppet_environment>/modules/is_common/files/packs` directory in the **Puppetmaster**.
```

Comment:
Why do we mention that the user should download an updated pack? This should work with the GA pack as well.

Reply:
Fixed https://github.com/wso2/puppet-is/pull/97/commits/de02671cf86cba30743acc2c6308e881922d866c